tests/code/test_events.py

```diff
@@ -389,6 +389,36 @@ def test_saveEventToDb_update():
 
         transaction.rollback()
 
+@pytest.mark.integration
+def test_cancelEvent():
+     with mainDB.atomic() as transaction:
+        # creates an event 
+        testingEvent = Event.create(name = "Testing canceled event",
+                                      term = 2,
+                                      description = "This Event is Created to be Canceled.",
+                                      timeStart = "07:00 PM",
+                                      timeEnd = "10:00 PM",
+                                      location = "Somewhere",
+                                      isRsvpRequired = 0,
+                                      isTraining = 0,
+                                      isService = 0,
+                                      startDate = "2021-12-12",
+                                      endDate = "2022-6-12",
+                                      isCanceled = False)
+        
+        testingEvent = Event.get(Event.name == "Testing canceled event")
+        eventId = testingEvent.id
+        
+
+        with app.app_context():
+            g.current_user = User.get_by_id("ramsayb2")
+            cancelEvent(eventId)
+            event = Event.get(Event.id == eventId)
+        assert event.isCanceled == True
```

Comment:
i'm too lazy to clone this repo but i'm sure you can just do `assert event.isCanceled 
`since it returns a boolean, it will evaluate correctly anyways

Reply:
Good catch!!!